pkg/proxy/winkernel/proxier_test.go

```diff
@@ -216,15 +148,12 @@ func TestCreateServiceVip(t *testing.T) {
 }
 
 func TestCreateRemoteEndpointOverlay(t *testing.T) {
```

Comment:
Thank you for adding all these tests. Can you please add comments on each of these test cases:
a) Providing a brief description of the testcase or the scenario it is testing
b) Partition the test case into 3 stages Arrange (how we are setting up the objects), Act (the method you are executing on the object), Assert  (testing the expected outcome), may be just by adding a comment //Arrange , //Act , //Assert before each stage